.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: release-chart
+on:
+  workflow_dispatch:    
+
+jobs:
+  release-chart:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Publish Helm chart
+        uses: stefanprodan/helm-gh-pages@v1.5.0
```

Comment:
Would it make more sense to use the official action from Helm?

https://github.com/helm/chart-releaser-action

Reply:
That would work as well, I was looking at #183 and decided to contribute based on it.

Reply:
i've updated the file to use the official action from the Helm team